WordPress/src/main/res/values/strings.xml

```diff
@@ -2253,8 +2253,9 @@
     <string name="quick_start_list_view_site_subtitle">@string/quick_start_dialog_view_site_message</string>
     <string name="quick_start_list_view_site_title">@string/quick_start_dialog_view_site_title</string>
     <string name="quick_start_sites">Quick Start</string>
-    <string name="quick_start_sites_progress" tools:ignore="UnusedResources" translatable="false">%1$d/%2$d</string>
-    <string name="quick_start_span_end" tools:ignore="UnusedResources" translatable="false">&lt;/span&gt;</string>
-    <string name="quick_start_span_start" tools:ignore="UnusedResources" translatable="false">&lt;span style=&quot;color:#78dcfa&quot;&gt;</string>
+    <string name="quick_start_sites_progress" translatable="false" tools:ignore="UnusedResources">%1$d/%2$d</string>
+    <string name="quick_start_span_end" translatable="false">&lt;/span&gt;</string>
+    <string name="quick_start_span_start" translatable="false">&#160;&lt;span style=&quot;color:#0&quot;&gt;</string> 
```

Comment:
Travis isn't liking the space at the end of this line.  Once that's removed, it should be good to merge!